lib/blockchain/install.js

```diff
@@ -117,5 +121,8 @@ function run () {
   })
 
   inquirer.prompt(questions)
-    .then(answers => processCryptos(answers.crypto))
+    .then(answers => {
+      answers.crypto.push(coinUtils.getCryptoCurrency(ETH_CRYPTO_CODE).code)
```

Comment:
limitation of Inquirer.js lib:
When a checkbox option is marked as disabled, it is not included on answers.